### DISCUSSION TITLE
Buy land to stop expand of the city

### DISCUSSION THREAD

**Discussion by filipchudoba** [voxeltycoon/issues#358] [General]
Sometimes it is annoying if i want to build factory and I have no space for it, so this can be a prevention.

**Comment by victorleblais** [comment]
How do you think this feature could be implemented in the current UI?
How would bought land look like?
Try explaining your idea in as much details as you can :)

As a workaround, you can build decorations top prevent the city from expanding, something like pumpkins would work.

**Comment by JezzyBelle** [comment]
I like this concept. I have seen companies buy large tracts of land like that outside of cities. Better to hold land than have to demolish homes and businesses for later expansion.
I am no programmer.. just a gamer. So here's my take on implementation.

Implementation: 
Purchasing land would be in blocks... IE: think the shape of your potential businesses and the footprint they need.  So, with that in mind, may need to be a few sizes. Image would be very similar to what territories we have not unlocked look like. IE: Border demarcation and also some shading. Otherwise, it would act just like a building. You would pay some limited regular fees for upkeep, etc until you utilize it. I also think there should be a timer. So, you only reserve it for a given time then it reverts and you suffer a penalty in the form of losing a deposit or something..... So, in effect, you do not own it. You just purchased an option to buy.

**Comment by hairintd** [comment]
you can build couple of cheap "decorations" which is kinda alternative. they are four now - two with size 1x1, one 1x3 and one expensive.
Also placing railways is a simple method when you have a lot of mon$y

**Comment by BiggsSkeedR** [comment]
@victorleblais, I would use the same UI and controls as terraforming, have it build fences around company owned land, perhaps with some shading to indicate it's owned. Costs would be cheap at the start and would follow a, probably slightly shallower, increase to the current inflation rate.

**Comment by singetail246** [comment]
Great idea, its just that besides the starting area, you've technically already bought the land, and for a good price too. So if decorations or tracks can already prohibit city growth, I'd say instead of requiring to buy the land again, just add some fence decorations or such that can be placed to make borders.

Just my take on your idea.
